docs/reference/esql/functions/count-distinct.asciidoc

```diff
@@ -30,11 +37,26 @@ properties:
 
 include::../../aggregations/metrics/cardinality-aggregation.asciidoc[tag=explanation]
 
-[discrete]
-==== Precision is configurable
-
 The `COUNT_DISTINCT` function takes an optional second parameter to configure the
-precision discussed previously.
+precision.
+
+*Supported types*
+
+Can take any field type as input and the result is always a `long` no matter the
+input type.
```

Comment:
This can stay as is/was, just finding it a bit curious that we bridge input and output types here, this being a count.
```suggestion
Can take any field type as input.
```